spartan/bootstrap.sh

```diff
@@ -62,7 +62,7 @@ function test_cmds {
     # echo "$hash timeout -v 30m ./spartan/bootstrap.sh test-kind-4epochs"
     # echo "$hash timeout -v 30m ./spartan/bootstrap.sh test-kind-upgrade-rollup-version"
     # echo "$hash timeout -v 30m ./spartan/bootstrap.sh test-prod-deployment"
-    echo "$hash ./spartan/bootstrap.sh test-cli-upgrade-with-lock"
+    echo "$hash ./spartan/bootstrap.sh test-cli-upgrade"
   fi
```

Comment:
If smoke takes 20m we are in serious trouble

Reply:
Is that happening now ?

Reply:
There was a diff where a timeout of 20 minutes was added to the smoke, that was concerning me